app/views/messages/outbox/create.js.erb

```diff
@@ -1,2 +1,3 @@
+<%# TODO: can we remove this? %>
```

Comment:
I don't think so, since it's updating the flash message. And there are many other similar files.

Reply:
Yeah but who renders this template? It may be dead code, there's also: app/views/provider/admin/messages/outbox/create.js.erb

Reply:
You're right, I don't know.

Reply:
I removed it in the end.. it probably was replaced by /provider/admin/messages/outbox/create.js.erb a long time ago.